src/components/STooltip.vue

```diff
@@ -117,7 +120,9 @@ export default class STooltip extends Vue {
   get tooltipStyle () {
     this.classList = this.callout ? 'callout' : 'tooltip'
     this.classList = this.offsetTopButton <= this.halfWindowHeight ? `${this.classList} bottom` : `${this.classList} top`
-    if (this.left || this.offsetLeftButton <= this.leftEdgeConstraint) {
+    if (this.middle) {
```

Comment:
if we specify several props like middle\left\right the behaviour would be unexpected. Maybe change it to one prop position with left/middle/right and error if something other?